docs/api/ShallowWrapper/hostNodes.md

```diff
@@ -0,0 +1,18 @@
+# `.hostNodes() => ShallowWrapper`
+
+Returns a new wrapper with only host nodes.
+
+
+
+#### Returns
+
+`ShallowWrapper`: A new wrapper that wraps the filtered nodes.
+
+
+
+#### Examples
+
+```jsx
+const wrapper = mount(<div><MyComponent className="foo" /><div className="foo" /></div>);
```

Comment:
should this be `shallow`?

Reply:
Whoops, good catch! Want to submit a PR?

Reply:
https://github.com/airbnb/enzyme/pull/1215